flake.nix

```diff
@@ -232,11 +227,34 @@
               gotools
               go-tools
               gofumpt
-            ];
+            ]);
             shellHook = ''
               ${pre-commit.shellHook}
             '';
           };
+
+          tf = {
+            inherit (pkgs) terraform opentofu;
+          };
```

Comment:
What is the motivation behind `tf`, rather than using `pkgs.terraform` and `pkgs.opentofu` directly?

Reply:
my idea had been to centralize them reasoning that it might make it easier to swap one of the two for a package from elsewhere, e.g. a newer version.
whether this actually achieves making them override-able i'm maybe not so sure of tho - so i'm okay with using them directly if preferred.